includes/qform_state_handlers/QFileFormStateHandler.class.php

```diff
@@ -86,7 +86,8 @@ public static function GarbageCollect() {
 			// Go through all the files
 			$objDirectory = dir(self::$StatePath);
 			while (($strFile = $objDirectory->read()) !== false) {
-				if (!count(self::$FileNamePrefix))
+				if (($strFile==".")||($strFile=="..")) continue;
+				if (!$FileNamePrefix)
```

Comment:
@willypowwow I think you made and error there. This should be self:$FileNamePrefix right?

Reply:
Yes, you are correct.  

```suggestion
				if (!$self::FileNamePrefix)
```